tests/examples_arguments_syntax/lasagna_plot.py

```diff
@@ -23,8 +23,8 @@
             format="%Y",
             labelAngle=0,
             labelOverlap=False,
-            labelColor=color_condition,
-            tickColor=color_condition,
+            labelColor=color_condition, # type: ignore[arg-type]
+            tickColor=color_condition, # type: ignore[arg-type]
```

Comment:
Related to https://github.com/vega/altair/pull/3544#discussion_r1745791727

Required different ignore comments, since these aren't for property method overload(s)

Reply:
Will be fixed by #3573